easybuild/easyconfigs/t/trame-bundle/trame-bundle-2025.03-foss-2024a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'trame-bundle'
+version = '2025.03'
+
+homepage = 'https://kitware.github.io/trame/'
+description = """Bundle of trame extensions"""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('trame', '3.8.0'),
+    # ('VTK', '9.3.1'),
```

Comment:
is trame_vtk replacing the this + trame_vtklocal? That isn't entirely clear, and it should be some comments explaining this if that is the case.

Reply:
No, trame_vtklocal is different from trame_vtk. If I understand correctly, trame_vtk is used to define vtk widgets and trame_vtklocal is to handle local rendering.
trame_vtk doesn't have vtk in its dependencies but trame_vtklocal does. However, trame_vtklocal requires vtk >=9.4, which is not the version in this toolchain and I am hesitate to add another version (and I am not sure whether trame_vtklocal is useful on clusters)